test/module/shared_model/command_mocks.hpp

```diff
@@ -32,48 +32,54 @@ using testing::Return;
 
 namespace shared_model {
   namespace interface {
-    struct MockCommand : public shared_model::interface::Command {
+    struct MockCommand : public Command {
       MOCK_CONST_METHOD0(get, const Command::CommandVariantType &());
     };
 
-    struct MockAddAssetQuantity
-        : public shared_model::interface::AddAssetQuantity {
+    struct SpecificMockCommandBase {};
+
+    template <typename T>
+    struct SpecificMockCommand : public T, public SpecificMockCommandBase {
+      using SpecificCommand = T;
```

Comment:
Please use 
https://github.com/hyperledger/iroha/blob/b01d1f8158ab6f24e1d90443c47118f4368fc022/shared_model/interfaces/base/model_primitive.hpp#L28 or https://github.com/hyperledger/iroha/blob/b01d1f8158ab6f24e1d90443c47118f4368fc022/shared_model/interfaces/base/noncopyable_model_primitive.hpp#L22 instead

Reply:
I removed this alias, but I still need the common ancestor class to distinguish a command from other `Model`s.